.github/workflows/macos.yml

```
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

Saw your post on mastodon. Thought this might help?

---

... nevermind, I misread, sorry. 🤦‍♂️